plugins/scroll-options/src/ScrollBlockDragger.js

```diff
@@ -84,6 +85,12 @@ export class ScrollBlockDragger extends Blockly.BlockDragger {
         Blockly.utils.Coordinate.sum(this.scrollDelta_, currentDragDeltaXY);
     super.drag(e, totalDelta);
     this.dragDelta_ = currentDragDeltaXY;
+
+    // Calculate the location the block is being dragged to, in ws units.
+    // This same calculation is done in super.drag().
+    const deltaPx = this.pixelsToWorkspaceUnits_(totalDelta);
+    const newLoc = Blockly.utils.Coordinate.sum(this.startXY_, deltaPx);
```

Comment:
I think this might point to us needing some kind of `getNewLocation` method in core.

Reply:
Cool, I'll do that next in core.

Reply:
Circling back, we decided offline not to do this in core since it needs both `delta` and `newLoc` so we'd just be wrapping the one line into a new function, which isn't super valuable here.